### DISCUSSION TITLE
Ability to mark selection as viewed

### DISCUSSION THREAD

**Discussion by bsrz** [community/community#64092] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

There are numerous times when you review code containing files with multiple chunks.
Having the ability to mark selections (the same type of selection when commenting) as viewed would be tremendously helpful in focusing the review (and subsequent commits) on the code that changed and hasn't been marked as viewed.

For example:

![](https://github.com/community/community/assets/1077441/ecac1643-de67-4aa5-82f3-cd652a1dbace)

**Comment by christophercouch** [comment]
Hi @bsrz,

I completely understand how marking specific selections as viewed can streamline the review process. This feature would indeed help in keeping track of reviewed sections, especially in large codebases or lengthy pull requests.

You might want to submit this as a feature request directly through the [GitHub feedback form](https://support.github.com/contact/feedback). The GitHub team often reviews feedback to improve and introduce new features.

Thanks for your valuable input!
